frappe/database/database.py

```diff
@@ -784,7 +776,7 @@ def _get_values_from_table(
 		limit=None,
 	):
 		field_objects = []
-
+		as_dict = fields == "*" and not isinstance(fields, (list, tuple, Criterion))
```

Comment:
This is replacing the passed argument and possibly reason behind CI failure.

Reply:
everything else LGTM.

Reply:
Ah, yes. I see why.

Reply:
![image](https://user-images.githubusercontent.com/9079960/183920131-60e02aec-42f6-4576-b49a-8ed8fd8514ae.png)


Previously we were only toggling it to true in some cases now we are toggling it to true in those cases ELSE making it false and totally ignoring the argument it was invoked with. That's the problem.

Reply:
![](https://www.memestemplates.com/wp-content/uploads/2022/06/you-made-those-words-up-scaled.jpg)